src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatabilityAnalyzer.cs

```diff
@@ -494,6 +526,43 @@ private static void ReportDiagnostics(IOperation operation, SmallDictionary<stri
                 _ => null,
             };
 
+        private static IEnumerable<ISymbol> GetPropertyAccessors(IPropertySymbol property, IOperation operation)
+        {
+            var usageInfo = operation.GetValueUsageInfo(property.ContainingSymbol);
+
+            // not checking/using ValueUsageInfo.Reference related values as property cannot be used as ref or out parameter
+            // not using ValueUsageInfo.Name too, it only use name of the property
+            if (usageInfo == ValueUsageInfo.ReadWrite)
+            {
+                yield return property.GetMethod;
+                yield return property.SetMethod;
+            }
+            else if (usageInfo.IsWrittenTo())
+            {
+                yield return property.SetMethod;
+            }
+            else if (usageInfo.IsReadFrom())
+            {
+                yield return property.GetMethod;
+            }
+            else
+            {
+                yield return property;
+            }
+        }
+
+        private static ISymbol GetEventAccessor(IEventSymbol iEvent, IOperation operation)
+        {
+            if (operation.Parent is IEventAssignmentOperation eventAssignment)
+            {
+                if (eventAssignment.Adds)
+                    return iEvent.AddMethod;
+                else
+                    return iEvent.RemoveMethod;
+            }
```

Comment:
Do we need to handle `RaiseMethod` as well? http://sourceroslyn.io/#Microsoft.CodeAnalysis/Symbols/IEventSymbol.cs,49.

Reply:
Oh well it is VB only :-) http://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp/Symbols/PublicModel/EventSymbol.cs,63. I am fine with not handling it here and just file a follow-up tracking issue.

Reply:
An issue created https://github.com/dotnet/roslyn-analyzers/issues/4089